test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,100 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc.
+
+    ```bash
+    cd test/scale-infra
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2 vpc_name=<aws-iam-user>"
```

Comment:
I still see "<aws-iam-user>" here, when I think you mean that this should be the same value identified as "<vpc_name>" in the next command. Please be consistent. I think that "vpc_name" is the better identifier, since it is more clearly a _choice_.

Reply:
ok, I see! Fixed, replaced `aws-iam-user` with `vpc_name` instead.